packages/material-ui-lab/src/MasonryItem/MasonryItem.test.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { createClientRender, describeConformanceV5 } from 'test/utils';
+import MasonryItem, { masonryItemClasses as classes } from '@material-ui/lab/MasonryItem';
+import { expect } from 'chai';
+import { createTheme } from '@material-ui/core/styles';
+import defaultTheme from '@material-ui/core/styles/defaultTheme';
+import ResizeObserver from 'resize-observer-polyfill';
+import { style } from './MasonryItem';
+
+// Mount ResizeObserver; otherwise, this error comes up: `[ReferenceError: ResizeObserver is not defined]`
+global.ResizeObserver = ResizeObserver;
```

Comment:
Let's move this into a `before` and clean it up in a `after`. Otherwise every subsequent test will have `ResizeObserver` being defined when that's not the default.

We probably also want to add a descriptive error when `ResizeObserver` isn't defined. It's not supported by all browsers we support so an error is generally unexpected.